src/test/java/org/prebid/server/bidder/adhese/AdheseBidderTest.java

```diff
@@ -106,6 +107,9 @@ public void makeHttpRequestsShouldModifyIncomingRequestAndSetExpectedHttpRequest
                         .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpAdhese.of("demo",
                                 "_adhese_prebid_demo_", "leaderboard", mapper.convertValue(targets, JsonNode.class)))))
                         .build()))
+                .device(Device.builder()
+                        .ifa("dum-my")
+                        .build())
```

Comment:
Minor. We prefer to use builder in separate lines only in case if single line is more then checkstyle line length rule constant.